var/spack/repos/builtin/packages/mysql/package.py

```diff
@@ -8,11 +8,31 @@
 class Mysql(CMakePackage):
     """MySQL is an open source relational database management system."""
 
-    homepage = "http://dev.mysql.com"
-    url      = "https://dev.mysql.com/get/Downloads/MySQL-8.0/mysql-8.0.11.tar.gz"
+    homepage = "https://www.mysql.com/"
+    url      = "https://dev.mysql.com/get/Downloads/MySQL-8.0/mysql-8.0.15.tar.gz"
 
+    version('8.0.15', sha256='bb1bca2dc2f23ee9dd395cc4db93b64561d4ac20b53be5d1dae563f7be64825e')
     version('8.0.11', '38d5a5c1a1eeed1129fec3a999aa5efd')
     version('5.7.22', '269935a8b72dcba2c774d8d63a8bd1dd')
 
-    depends_on('boost@1.66.0', when='@8.0.11')
-    depends_on('boost@1.59.0', when='@5.7.22')
+    # https://dev.mysql.com/doc/refman/8.0/en/source-installation.html
+
+    # See CMAKE_MINIMUM_REQUIRED in CMakeLists.txt
+    depends_on('cmake@3.8.0:', type='build', when='platform=win32')
+    depends_on('cmake@3.9.2:', type='build', when='platform=darwin')
+    depends_on('cmake@3.4.0:', type='build', when='platform=solaris')
```

Comment:
No clue what platform Solaris/SunOS actually reports as, but this is the CMake version required.